manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3rc

```

Comment:
```suggestion
PackageVersion: 1.1.3
```

Reply:
Hi @ItzLevvie!

You're talking about this version number?
![grafik](https://user-images.githubusercontent.com/87755490/126485948-17c7262f-cd91-4d9f-a4d8-0cee86803729.png)
That's an error in the installer. Thanks for mentioning it. Should also display 1.1.3rc

Reply:
I updated the installer and the SHA. Should now work as intended